providers/base/units/stress/s3s4.pxu

```diff
@@ -150,3 +150,221 @@ requires: cpuinfo.platform in ("i386", "x86_64")
 command: [ -e "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log ] && cat "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log
 _description:
  Attaches the log from the {s4_iterations} cycles Hibernate/Resume test if it exists
+
+id: stress_s3_cycles_iterations_1
+category_id: stress-tests/suspend
+summary:
+  Resource job meant to be used in the S3 cycles stress templates
```

Comment:
This should say what it does not how some other job uses it.
"This resource generates suspend_reboot iteration numbers"

Reply:
ok~~ will correct all resource jobs' summary